pom.xml

```diff
@@ -96,6 +101,22 @@
   </dependencyManagement>
 
   <build>
+
+	<plugins>
+		 <plugin>
+			<groupId>org.apache.maven.plugins</groupId>
+			<artifactId>maven-compiler-plugin</artifactId>
+			<version>3.8.0</version>
```

Comment:
Since `oss-parent` defines `version.plugin.compiler`, better to override that in `properties`, refer here?
Or alternatively we can increase baseline in `oss-parent` (https://github.com/FasterXML/oss-parent) version `34`.